lerna.json

```diff
@@ -0,0 +1,7 @@
+{
+  "lerna": "2.11.0",
```

Comment:
Can we use the latest version here? https://www.npmjs.com/package/lerna

Reply:
We pull in `"lerna": "^3.7.1"` in the devDependencies. We should unify these at the very least, right?

Reply:
I switched this to the latest version and also started a PR to upgrade it for `opencensus-node` as well: https://github.com/census-instrumentation/opencensus-node/pull/263

Reply:
Realized I misunderstood the comment about unification that this involve fixing `lerna.json` here not necessarily upgrading it in `opencensus-node`. Fixed it here to match the `3.8.5` in `devDependencies`.